lntest/itest/lnd_macaroons_test.go

```diff
@@ -36,14 +36,14 @@ func testMacaroonAuthentication(net *lntest.NetworkHarness, t *harnessTest) {
 		}
 		testNode = net.Alice
 	)
+	ctxt, cancel := context.WithTimeout(ctxb, defaultTimeout)
```

Comment:
So you're sure this won't introduce in any new flakes? I guess given all these operations are just "shallow RPC" actions, we shouldn't run into the typical delays that can happen when mempool/blocks are involved.

Reply:
Yeah, pretty sure. None of these actions have any reason to take longer than a few milliseconds each. We never have to wait for any chain or mempool state.